production-swarm/docker-stack.yml

```diff
@@ -3,20 +3,15 @@ services:
   jobservice:
     env_file:
       - ./rabbitmq.env
-      - ./postgres.env
     environment:
+      CAF_DATABASE_URL: "jdbc:postgresql://${JOBSERVICE_DB_HOST:-192.168.56.10}:${JOBSERVICE_DB_PORT:-5432}/jobservice"
       CAF_DATABASE_USERNAME: "${CAF_DATABASE_USERNAME:-postgres}"
       CAF_DATABASE_PASSWORD: "${CAF_DATABASE_PASSWORD:-root}"
       CAF_STATUS_CHECK_TIME: 5
       CAF_TRACKING_PIPE: jobtracking-in
-      CAF_WEBSERVICE_URL: http://jobservice:9411/job-service/v1
-    image: jobservice/job-service:2.1.0
-    deploy:
-      restart_policy:
-        condition: any
-        delay: 5s
-        max_attempts: 3
-        window: 120s    
+      CAF_WEBSERVICE_URL: "http://jobservice:${JOB_SERVICE_PORT:-9411}/job-service/v1"
```

Comment:
Since this is the external port `jobservice` doesn't make sense as the hostname here - it should be the host where the swarm is running.